def test_failure():
    assert False
